tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -52,12 +64,23 @@ Verify Builds Number
   Should Be Equal As Integers  ${build_length}  ${expected_builds}
   [Return]  ${builds}
 
-Verify Builds Status 
+Verify Builds Status
   [Arguments]  ${build_status}
   @{builds}=  Oc Get  kind=Build  namespace=redhat-ods-applications
   FOR  ${build}  IN  @{builds}
-    Should Be Equal As Strings  ${build}[status][phase]  ${build_status}  
+    Should Be Equal As Strings  ${build}[status][phase]  ${build_status}
     Should Not Be Equal As Strings  ${build}[status][phase]  Cancelled
     Should Not Be Equal As Strings  ${build}[status][phase]  Failed
     Should Not Be Equal As Strings  ${build}[status][phase]  Error
-  END
\ No newline at end of file
+  END
+
+Install RHODS on OSD Cluster
```

Comment:
The catalogsource installation is the same in PSI/OSD

Reply:
I am aware of that just that I didn't want to disturb what was already present so i added the condition. However i removed the OSD part